gradle/manifest-gen.gradle

```diff
@@ -0,0 +1,74 @@
+/******************************************************************************
+ * Copyright (c) 2016 TypeFox and others.
+ * 
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0,
+ * or the Eclipse Distribution License v. 1.0 which is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ * 
+ * SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
+ ******************************************************************************/
+
+apply plugin: 'osgi'
+
+def baseVersion = project.version
+if (baseVersion.endsWith('-SNAPSHOT'))
+	baseVersion = baseVersion.substring(0, baseVersion.length() - 9)
+else if (baseVersion.split('\\.').length == 4)
+	baseVersion = baseVersion.substring(0, baseVersion.lastIndexOf('.'))
+def qualifiedVersion = baseVersion + '.v' + buildTime
+
+jar.manifest {
+	version = qualifiedVersion
+	if (project.hasProperty('title'))
+		name = project.title
```

Comment:
There seems to be a bug in the build config from LSP4J. Please remove lines 24 and 25 and add this block after the `jar.manifest` block instead:
```groovy
afterEvaluate {
	if (project.hasProperty('title'))
		jar.manifest.name = project.title
}
```

Reply:
Ok that does seem to do the trick!